Use configured value instead of hardcoded path

diff -up mariadb-10.0.13/scripts/mysql_install_db.pl.in.pbasedir mariadb-10.0.13/scripts/mysql_install_db.pl.in
--- mariadb-10.0.13/scripts/mysql_install_db.pl.in.pbasedir	2014-09-04 12:50:24.061979080 +0200
+++ mariadb-10.0.13/scripts/mysql_install_db.pl.in	2014-09-04 12:51:22.929045559 +0200
@@ -318,7 +318,7 @@ elsif ( $opt->{basedir} )
                     find_in_basedir($opt,"file","mysqld-nt",
                                   "bin");  # ,"sql"
   $pkgdatadir     = find_in_basedir($opt,"dir","fill_help_tables.sql",
-                                    "share","share/mysql");  # ,"scripts"
+                                    "share","@INSTALL_MYSQLSHAREDIR@");  # ,"scripts"
   $scriptdir      = "$opt->{basedir}/scripts";
 }
 else
